content/docs/user-guide/build/troubleshooting.md

```diff
@@ -130,20 +130,32 @@ time cmake --build build/linux --target Editor AutomatedTesting.GameLauncher --c
 1. Make sure you are only building the targets you are using.  For example, select an appropriate target like the `Editor` for building and not the `ALL_BUILD` target which will build everything and take longer.
   Keep in mind that your first build will take the longest because all the code for the selected target and dependencies must be compiled, but subsequent builds should be much faster (minutes or seconds) especially when writing game code.
 1. Use the `profile` build configuration when developing when you don't need all the un-optimized debugging information of a `debug` build configuration.
+1. Make sure you're using unity builds (on by default) by configuring with the option `-DLY_UNITY_BUILD=ON` build configuration when developing when you don't need all the un-optimized debugging information of a `debug` build configuration.
```

Comment:
It looks like the "build configuration when developing when you don't need all the un-optimized debugging information of a `debug` build configuration" blurb tagged along from the previous item in the list here

Reply:
good catch!